docs/changelog.rst

```diff
@@ -30,6 +30,9 @@ Major new features:
 * :doc:`/plugins/albumtypes`: An accompanying plugin for formatting
   ``albumtypes``. Thanks to :user:`edgars-supe`.
 
+* Add new VGMdb source for match and fetchart
+  :user:`eldarock`.
```

Comment:
Seems like it should probably be two bullet points: one for the new plugin and one for the new feature in `fetchart`.